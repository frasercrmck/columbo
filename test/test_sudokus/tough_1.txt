# RUN: columbo -q -f %s
# From sudokuwiki.org, Tough Example
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF
0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF 0x1FF

13 A01 B1
11 A23
12 AB4
9  AB5
12 AB6
8  AB7
24 ABCD8
18 BCDE0
11 B23
14 CDE1
15 C234
21 C567 D7
10 D234
20 D56 E56
30 E23 F23
18 E4 F456
8  EFG7
20 EFGH8
24 FGHJ0
11 FG1 G23
24 G456
15 HJ1
11 HJ2
6  HJ3
10 HJ4
8  H56
7  J56
15 H7 J78
