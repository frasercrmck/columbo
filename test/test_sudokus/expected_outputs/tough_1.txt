0x004 0x080 0x010 0x020 0x100 0x001 0x040 0x002 0x008 
0x001 0x002 0x008 0x040 0x004 0x080 0x010 0x020 0x100 
0x020 0x040 0x100 0x002 0x008 0x010 0x001 0x080 0x004 
0x100 0x004 0x001 0x008 0x010 0x002 0x020 0x040 0x080 
0x002 0x008 0x040 0x080 0x020 0x004 0x100 0x001 0x010 
0x080 0x010 0x020 0x100 0x001 0x040 0x008 0x004 0x002 
0x010 0x001 0x002 0x004 0x040 0x100 0x080 0x008 0x020 
0x008 0x100 0x004 0x001 0x080 0x020 0x002 0x010 0x040 
0x040 0x020 0x080 0x010 0x002 0x008 0x004 0x100 0x001 

13 A0 A1 B1
11 A2 A3
12 A4 B4
9  A5 B5
12 A6 B6
8  A7 B7
24 A8 B8 C8 D8
18 B0 C0 D0 E0
11 B2 B3
14 C1 D1 E1
15 C2 C3 C4
21 C5 C6 C7 D7
10 D2 D3 D4
20 D5 D6 E5 E6
30 E2 E3 F2 F3
18 E4 F4 F5 F6
8  E7 F7 G7
20 E8 F8 G8 H8
24 F0 G0 H0 J0
11 F1 G1 G2 G3
24 G4 G5 G6
15 H1 J1
11 H2 J2
6  H3 J3
10 H4 J4
8  H5 H6
7  J5 J6
15 H7 J7 J8
